src/request.ts

```diff
@@ -284,6 +285,16 @@ class DatastoreRequest {
           readConsistency: code,
         };
       }
+      if (options.readTime) {
+        if (!reqOpts.readOptions) {
+          reqOpts.readOptions = {};
+        }
+        const readTime = options.readTime;
+        const seconds = readTime / 1000;
```

Comment:
Could you just make this `parseInt(readTime / 1000`, rather than having the floor? (I'm guessing this type should be an integer?

Reply:
`parseInt` only accepts string as a parameter. Are you sure about this? Should I convert it into a string and then use parseInt?